README.md

```diff
@@ -144,8 +144,16 @@ such as restricting discovered features with the --label-whitelist option._
 
 ### Usage
 
-Feature discovery is done as a one-shot job. There is an example script in this
-repo that demonstrates how to deploy the job to unlabeled nodes.
+Feature discovery is preferably run as a Kubernetes DaemonSet. There is an
+example spec that can be used as a template, or, as is for trying out the
+service:
+```
+kubectl create -f node-feature-discovery-daemonset.json.template
+```
```

Comment:
Can you add a blurb on how to change the sleep interval? Something along the lines of:
`When run as a daemonset, the node is relabelled at an interval specified using the --sleep-interval flag. The default interval is set to 60s.`

May be we can add this arg in the daemonset template and add a link to the line in the above documentation as well?

Reply:
> add a link to the line in the above documentation as well?

JSON does not allow comments, afaik